x-pack/plugins/security_solution/public/common/components/popover_items/index.test.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { PopoverItems, PopoverItemsProps } from '.';
+import { TestProviders } from '../../mock';
+import { render, screen } from '@testing-library/react';
+import { within } from '@testing-library/dom';
+
+const mockTags = ['Elastic', 'Endpoint', 'Data Protection', 'ML', 'Continuous Monitoring'];
+
+const renderHelper = (props: Partial<PopoverItemsProps<unknown>> = {}) =>
+  render(
+    <TestProviders>
+      <PopoverItems
+        dataTestPrefix="tags"
+        items={mockTags}
+        popoverButtonTitle="show mocks"
+        renderItem={
+          ((item: string, index: number) => (
+            <span key={`${item}-${index}`}>{item}</span>
+          )) as PopoverItemsProps<unknown>['renderItem']
```

Comment:
Do we still need this `as PopoverItemsProps<unknown>['renderItem']`?

Reply:
oh, sorry, missed that one in tests.
Removed